fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCQueryBuilder.java

```diff
@@ -925,13 +908,10 @@ private SqlQueryData processDateParm(Class<?> resourceType, QueryParameter query
         final String METHODNAME = "processDateParm";
         log.entering(CLASSNAME, METHODNAME, queryParm.toString());
 
-        boolean isDateSearch = isDateSearch(resourceType, queryParm);
-        boolean isDateRangeSearch = isDateRangeSearch(resourceType, queryParm);
-
-        if (log.isLoggable(Level.FINE)) {
-            log.fine("isDateSearch=" + isDateSearch + "  isDateRangeSearch=" + isDateRangeSearch);
-        }
-        if (!isDateSearch && !isDateRangeSearch) {
+        // Check to see if this is of a type is date. 
+        String code = queryParm.getCode();
+        SearchParamType type = SearchUtil.getSearchParameter(resourceType, code).getType();
+        if (!SearchParamType.DATE.getValue().equals(type.getValue())) {
```

Comment:
Is this needed?  How would we get here if the SearchParamType isn't date?

Reply:
Good point, I'm not sure where this originated. It's definitely not needed.